docs/tutorial_langcheckchat.md

```diff
@@ -0,0 +1,125 @@
+
+Below is a step-by-step process on how to use the app.
+
+### Step 0: Get the app up and running
+To get the app up and running, you will need to:
+1. [Clone the repo and install requirements](https://github.com/citadel-ai/langcheckchat?tab=readme-ov-file#installation)
+2. [Update the environment variables with your OpenAI API details](https://github.com/citadel-ai/langcheckchat?tab=readme-ov-file#1-update-the-environment-variables-with-your-openai-api-details)
+3. [Run the app](https://github.com/citadel-ai/langcheckchat?tab=readme-ov-file#2-run-the-app)
+
+Once the app is running, you should see a page that looks like this:
+
+![LangCheckChat 1](_static/LangCheckChat_1.png)
+
+### Step 1: Ask a question
+Let’s now try asking a question! Type in “how can I check for hallucinations?” (or anything else you want) and hit enter.
+
+![LangCheckChat 2](_static/LangCheckChat_2.png)
+
+### Step 2: Check the results and evaluation metrics
+Once the question is submitted, you should first see the RAG system’s response show up, and then see the various LangCheck metrics being computed. Once everything has been computed, you should see the following:
+- ***Prompt***: the question you asked in step 1
+- ***Answer***: the RAG system’s final answer to your question
+- ***Source Document***: the source(s) retrieved by the RAG system
+- [***Reference-Free Text Quality Metrics***](https://langcheck.readthedocs.io/en/latest/langcheck.metrics.en.reference_free_text_quality.html): LangCheck metrics that can be computed without the source or reference texts
+- [***Source-Based Text Quality Metrics***](https://langcheck.readthedocs.io/en/latest/langcheck.metrics.en.source_based_text_quality.html): LangCheck metrics based on the source text
+- ***Metric Explanations***: Some metrics (the ones that have the question mark icon) also have explanations for why the metric was given a certain score. Hover over the icon to see the explanation.
+
+### Step 3: (Optional) Enter a reference answer
+Optionally, if you know what the answer should be to your question, you can enter the reference answer.
+
+![LangCheckChat 3](_static/LangCheckChat_3.png)
+
+### Step 4: (Optional) Check the reference-based metric results
+If you entered a reference answer as outlined in step 3, you should see a new metrics table called [Reference-Based Text Quality Metrics](https://langcheck.readthedocs.io/en/latest/langcheck.metrics.en.reference_based_text_quality.html). These metrics compute how similar the LLM’s answer is to the reference text in various ways.
+
+![LangCheckChat 4](_static/LangCheckChat_4.png)
+
+### Step 5: (Optional) Check the logs of past interactions
+At the bottom of the page, there’s a link that says “See Q&A Logs”, and clicking that will take you to the logs page. You should see your latest interaction with LangCheckChat in the logs table, and all future interactions will be similarly tracked in this table.
+
+![LangCheckChat 5](_static/LangCheckChat_5.png)
```

Comment:
Might be good to have screenshots of a hallucination or otherwise bad answer and link to the code that runs the metric and adds the warning

Reply:
Added a screenshot in https://github.com/citadel-ai/langcheck/pull/95/commits/dc588e55d0c9d1b39fce89a31c4a163d76ef16ee

Reply:
Thanks! I added a sentence linking to the actual threshold since it's a useful reference